.github/workflows/ci.yml

```diff
@@ -83,4 +83,59 @@ jobs:
           name: mailparse-${{matrix.version}}-${{matrix.arch}}-${{matrix.ts}}
           path: binaries
       - name: test
-        run: nmake test TESTS="--show-diff tests"
\ No newline at end of file
+        run: nmake test TESTS="--show-diff tests"
+  windows-php-8-4:
```

Comment:
Why is it necessary to duplicate the code for PHP 8.4? Couldn't the matrix above just be extended?

Reply:
My bad.
I thought php 8.2 and 8.3 have to run with Windows 2019.

Reply:
That is indeed somewhat confusing. PHP 8.0 - 8.3 are supposed to be built with VS 2019, but 8.4 is supposed to be built with VS 2022. Now, windows-2019 ships VS 2019, and windows-2022 ships VS 2022. However, it is possible to install the VS 2019 toolset for VS 2022, and that is done by default on windows-2022 runners, so PHP 8.0 - 8.3 can be built there as well.

However, PHP 7.2 - 7.4 are supposed to be built with VS 2017, and even that toolset can be installed for VS 2022, but it is no longer by default. Therefore these PHP versions one would need to install the VS 2017 toolset in VS 2022 (takes a long time), or to use windows 2019 runners (more complex matrix setup). Unfortunately, it is not possible to install the VS 2022 toolset for VS 2019.

Reply:
I also tried to add php 8.0 and 8.1 but got the following error:
https://github.com/reeperbahnause/pecl-mail-mailparse/actions/runs/10623195080/job/29449120416  

This is probably because of the `--enable-test-ini` switch.
Looks like it is not available prior to php 8.2?

Reply:
D'oh! By introducing the `--enable-test-ini` option for PHP 8.2, I broke the simple solution of loading mbstring.dll.

Now we probably need conditional steps (Build mailparse for Windows, patch test ini, test). :(